docs/release.md

```diff
@@ -0,0 +1,167 @@
+
+# Kids First FHIR Model Release Specification
+
+## Background
+In software development a release is usually a package of source code with a
+version, which identifies a logical set of content having specific compatibility
+characteristics. A software release is used to distribute code to end users.
+
+In FHIR model development a release is very similar to a software release. It
+is a logical set of conformance resources, distributed in a package with a
+version that identifies the set of resources and their compatibility characteristics.
+
+The FHIR specification includes a
+[packaging specification](https://confluence.hl7.org/display/FHIR/NPM+Package+Specification)
+which uses NPM packages to distribute the model. When the IG publisher runs
+it produces the NPM package in the user's FHIR package cache at `~/.fhir/packages`
+containing the conformance resources from the FHIR model.
+
+<p align="center">
+<img src="images/npm.png" alt="FHIR NPM" height="200"/>
+</p>
+
+The FHIR packaging specification is still under development and does not have
+a fully fleshed out release process, but we can still use the versioning,
+package naming, etc. guidelines in the specification as a basis for the
+Kids First release specification.
+
+## Versioning
+We will follow the FHIR NPM package specification's versioning scheme which
+uses [Semantic Versioning](https://semver.org/) to determine version numbers
+for FHIR model releases:
+
+<img src="images/semvar.png" alt="Semantic Versioning Example" height="250"/>
+
+### Examples
+
+1. Create first FHIR model release        -> version `0.1.0`
+2. Create new StructureDefinition(s)      -> version `0.2.0`
+3. Remove a StructureDefinition           -> version `1.0.0`
+4. Change StructureDefinition.id          -> version `2.0.0`
+5. Fix typo in StructureDefinition.title  -> version `2.0.1`
+
+## Release Overview
+
+For Kids First, a release of the FHIR model will consist of a little more than
+the NPM package. Here is an overview:
+
+<p align="center">
+<img src="images/release.png" alt="FHIR Model Release"/>
+</p>
+
+Kids First will use Github releases and tags on a `release` branch to manage
+FHIR model releases.
+
+### Testing
+
+The steps in the [Pre-Release](#Pre-Release) and
+[Release Process](#Release-Process) sections include a rigorous testing process
+which includes unit tests and integration tests (read further for details) so
+that when the FHIR model is deployed to the production FHIR server, there is a
```

Comment:
```suggestion
that there is a
```

Reply:
a comma after the clause requires a comma before the clause too. I moved it to the end of the sentence instead.